File: consecutivos.c
#include <stdio.h>

int main(){

    int n;
    int quantidade = 1;
    int resultado = 1;

    scanf("%d", &n);
    int num[n];

    scanf("%d", &num[0]);
    for(int i = 1; i < n; i++){
        scanf("%d", &num[i]);
        if(num[i] == num[i - 1]){
            quantidade++;   
        } else {
            if(quantidade >= resultado){
            resultado = quantidade;
            } 
            quantidade = 1;
        }
    }
    if(quantidade > resultado) resultado = quantidade;
    
    printf("%d\n", resultado);

    return 0;
}

File: consecutivos.py
n = int(input())
nums = list(map(int, input().split()))

quantidade = 1
resultado = 1

for i in range(1, n):
    if nums[i] == nums[i - 1]:
        quantidade += 1
    else:
        if quantidade >= resultado:
            resultado = quantidade
        quantidade = 1

if quantidade > resultado:
    resultado = quantidade

print(resultado)